mise.toml

```diff
@@ -0,0 +1,11 @@
+[tools]
+"npm:xo-cli" = "0.32.1"
+python = "3.13"
+uv = "0.6"
```

Comment:
is it `mise` that relies on `uv` for the multiple python interpreter support?

Reply:
`mise` use the same python binaries as `uv` but it doesn't require `uv` to install python.
`uv` is there to deal with the python dependencies